app/controllers/application_controller.rb

```diff
@@ -145,9 +145,11 @@ def action_permission
 
   # not all models includes Authorizable so we detect whether we should apply authorized scope or not
   def resource_base
-    @resource_base ||= model_of_controller.respond_to?(:authorized) ?
-        model_of_controller.authorized(current_permission) :
-        model_of_controller.scoped
+    if model_of_controller.respond_to?(:authorized)
+      @resource_base ||= model_of_controller.authorized(current_permission)
```

Comment:
The same comment applies to almost all of these changes, but:

In most cases you could remove some duplication by doing: `@resource_base ||= if ...` instead of assignment within the conditional.